python/dataproc_templates/util/template_constants.py

```diff
@@ -16,10 +16,24 @@
 PROJECT_ID_PROP = "project.id"
 
 # Data
+TEXT_HEADER = "header"
+TEXT_INFER_SCHEMA = "inferSchema"
```

Comment:
We can combine CSV and TXT to one. Instead of using TEXT_HEADER, TEXT_INFER_SCHEMA and CSV_HEADER, CSV_INFER_SCHEMA we can keep common variables as HEADER and INFER_SCHEMA. We would require to make changes to GCS_TO_BigQuery and GCS_TO_BigTable templates for reading csv.

Reply:
Combined duplicate constants and updated GCS_To_BigQuery, GCS_To_BigTable and Text_To_BigQuery templates.